common/v2/features/Home/Home.tsx

```diff
@@ -38,20 +37,7 @@ const Section = styled.section`
   justify-content: center;
 `;
 
-// Hack to enforce specificity of our style on base component.
-// https://github.com/styled-components/styled-components/issues/1816#issuecomment-398454088
-const SLayout = styled(Layout)`
-  && {
-    background-color: #fff;
-  }
-`;
-
-const BottomSection = styled(Section)`
-  @media (max-width: ${SCREEN_SM}) {
-    flex-direction: column;
-    align-items: center;
-  }
-`;
```

Comment:
This section needs `flex-direction: column` to display ok on smaller screens.

without:
![image](https://user-images.githubusercontent.com/12676806/66302770-d33cf900-e8f9-11e9-8534-2f4ccc992324.png)

with:
![image](https://user-images.githubusercontent.com/12676806/66302919-23b45680-e8fa-11e9-88a1-60ff448bbbb7.png)

Reply:
Fixed. Thanks!